docs/mp/lra/01_introduction.adoc

```diff
@@ -0,0 +1,115 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+= About Long Running Activities (LRA) in Helidon MP
+:h1Prefix: MP
+:pagename: intro-lra-in-mp
+:description: Introduction to LRA in Helidon MP
+:keywords: helidon, java, lra, mp, microprofile
+:javadoc-base-url-api: {javadoc-base-url}io.helidon.microprofile.cors/io/helidon/microprofile/lra
+:helidon-tag: https://github.com/oracle/helidon/tree/{helidon-version}
+:quickstart-example: {helidon-tag}/examples/quickstarts/helidon-quickstart-mp
+:lra-spec: https://github.com/eclipse/microprofile-lra/blob/master/spec/src/main/asciidoc/microprofile-lra-spec.adoc
+:helidon-mp-lra-example: {helidon-tag}/examples/microprofile/lra
+:common-page-prefix-inc: ../../shared/lra/common_shared.adoc
+:mp-pages-ref-prefix: mp/lra
+:mp-using-lra-ref: {mp-pages-ref-prefix}/02_using-lra.adoc
+:mp-lra-config-ref: {mp-pages-ref-prefix}/03_configuration-with-lra-mp.adoc
+:helidon-variant: MP
+
+Microservices create challenges for data consistency and integrity that necessitate changes in the transaction processing and data patterns used by them. Helidon MP now has its own LRA implementation that adds additional features to the existing MicroProfile LRA specification.
```

Comment:
Is "LRA" a well-known acronym to readers of this?  (I know that my own brain wants to substitute "Least Recently…" and then I can't complete the "A". 😄 )

Reply:
I see that it's in the title, but depending on where and when the title is rendered and on what platforms and such you may want to use "LRA (Long Running Activities)" instead to start.

Reply:
And to that point it's actually Long Running Actions. That was my mistake (older specs called them Long Running Activities and old habits die hard).